src/main/rx/cc2500_frsky_shared.c

```diff
@@ -207,7 +290,9 @@ static bool tuneRx(uint8_t *packet)
     }
     if (cc2500getGdo()) {
         uint8_t ccLen = cc2500ReadReg(CC2500_3B_RXBYTES | CC2500_READ_BURST) & 0x7F;
-        if (ccLen) {
+        if (ccLen > RX_SPI_MAX_PAYLOAD_SIZE) {
+            cc2500Strobe(CC2500_SFRX);
```

Comment:
I tested on a Taranis and Deviation.  I can pull it out if you don't like it.  There was some situations where `ccLen` was larger then `RX_SPI_MAX_PAYLOAD_SIZE` causing a lockup because it went over the buffer. I had figured the extra safety was worth the check.

Reply:
I can see value in this change, so I'd leave it in - we'll just have to make sure that it is available to testers to get some exposure before it gets into a release.

I think it's probably best if we wait with mergeing this until after 4.0 is out, this will give us plenty of time and exposure.